src/lib/modernExtend.ts

```diff
@@ -1502,14 +1502,17 @@ export function binary(args: BinaryArgs): ModernExtend {
     return {exposes: [expose], fromZigbee, toZigbee, configure, isModernExtend: true};
 }
 
+// eslint-disable-next-line
+export type Getter = (msg: Fz.Message, attributeKey: string | number) => any;
```

Comment:
```suggestion
export type Parse = (msg: Fz.Message, attributeKey: string | number) => unknown;
```

- After this you can remove `// eslint-disable-next-line` I thnk
- Can you rename `getter to `parse`?

Reply:
done